tests/integration/tests/test_sanity_checks.py

```diff
@@ -116,3 +127,26 @@ def test_cnvm_findings(cnvm_client, match_type):
     query, sort = cnvm_client.build_es_must_match_query(must_query_list=query_list, time_range="now-24h")
     results = get_findings(cnvm_client, CNVM_CONFIG_TIMEOUT, query, sort, match_type)
     assert len(results) > 0, f"The resource type '{match_type}' is missing"
+
+
+@pytest.mark.sanity
+@pytest.mark.parametrize("match_type", tests_data["cis_gcp"])
+def test_cspm_g_c_p_findings(cspm_client, match_type):
```

Comment:
would be easier to find through search
```suggestion
def test_cspm_gcp_findings(cspm_client, match_type):
```